vagrant/roles/logs.collect/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create log directory structure
+  file:
+    path: "{{ config.logdir }}/{{ inventory_hostname }}"
+    state: directory
+  delegate_to: localhost
+
+- name: Process backend specific tasks
+  include_role:
+    name: 'sit.{{ config.be.name }}'
+    tasks_from: cluster/main.yml
```

Comment:
I think that something like `collect` instead of `cluster` would be more meaningful.

Reply:
I've split the tasks into `logs` and `config`.